### ISSUE TITLE
Infrastructure data cannot be properly be imported

### ISSUE THREAD

**Issue by kwa20** [hzi-braunschweig/SORMAS-Project#5293]
### Bug Description
When .csv files are imported, data is either missing, replaced or can be imported with the wrong templates. This makes it almost impossible to set up a server as desired and verify a consistent import of data.

### Steps to Reproduce

1. Import the .csv file provided below in an empty region table
2. Check entries
3. Perform another import with the same file with the overwrite checkbox checked or unchecked

### Expected Behavior
Infrastructure data should only be importable with the correct template. Names for data should not be treated as duplicates when it's not an exact match. 

### Screenshots
<img width="584" alt="infrastructureImportError" src="https://user-images.githubusercontent.com/70317594/116738009-2f8ece80-a9f2-11eb-99e1-034aa8e333c8.PNG">
<img width="209" alt="infrastructureImportError2" src="https://user-images.githubusercontent.com/70317594/116737980-27369380-a9f2-11eb-971e-eb0cab128755.PNG">
Import of default countries misses one entry, even though the import was seemingly successful. Import result could not be seen due to #4905 

---

<img width="428" alt="infrastructureImportError4" src="https://user-images.githubusercontent.com/70317594/116737794-f2c2d780-a9f1-11eb-9693-76081bcec2f5.PNG">
<img width="616" alt="infrastructureImportError5" src="https://user-images.githubusercontent.com/70317594/116737803-f48c9b00-a9f1-11eb-9440-02cb733f92e4.PNG">
<img width="840" alt="infrastructureImportError3" src="https://user-images.githubusercontent.com/70317594/116737785-f0f91400-a9f1-11eb-935a-ad7b1586d514.PNG">
"Sachsen" is missing or can't be imported because it apparently already exists. However, the database was previously empty.

---

<img width="432" alt="infrastructureImportError6" src="https://user-images.githubusercontent.com/70317594/116737806-f5bdc800-a9f1-11eb-954d-605ce378a9f4.PNG">
<img width="492" alt="infrastructureImportError7" src="https://user-images.githubusercontent.com/70317594/116737808-f6eef500-a9f1-11eb-8b03-54adfb682575.PNG">
<img width="478" alt="infrastructureImportError8" src="https://user-images.githubusercontent.com/70317594/116737813-f8b8b880-a9f1-11eb-93de-6b05597336a9.PNG">

Re-imports of the same file have different results.

---

![2021-04-30_17-04-1619797376](https://user-images.githubusercontent.com/70317594/116739170-a7112d80-a9f3-11eb-961a-c72f687b6d79.jpg)

That is the result of three subsequent imports. However, even though "Sachsen" now exists only once, associated districts cannot be imported because "Sachsen" is apparently not a unique entry. 

<img width="917" alt="infrastructureImportError13" src="https://user-images.githubusercontent.com/70317594/116741604-a0d08080-a9f6-11eb-8d97-109b777b99fe.PNG">

---

<img width="427" alt="infrastructureImportError12" src="https://user-images.githubusercontent.com/70317594/116737832-fe160300-a9f1-11eb-85dd-1d30636edd6e.PNG">

districts can successfully be imported into regions.

### System Details

* Device:
* SORMAS version: 1.59.0
* Android version/Browser: firefox
* Server URL: survnet-import-test.sormas.netzlink.com
* User Role: admin, national user

### Additional Information

[sormas_bundeslaender_master_txt.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/6407406/sormas_bundeslaender_master_txt.txt)
[sormas_landkreise_master_txt.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/6407410/sormas_landkreise_master_txt.txt)

**Comment by MateStrysewske** [comment]
Duplicate of #4994
